receiver/prometheusreceiver/config.go

```diff
@@ -20,6 +20,9 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusreceiver/targetallocator"
 )
 
+// EscapedCaptureGroupOne is a constant that gets converted to "$1" during config preprocessing
+const EscapedCaptureGroupOne = "__capture_group_1__"
```

Comment:
Does this need to be exported? Seems like it isn't used in any other package.
```suggestion
const escapedCaptureGroupOne = "__capture_group_1__"
```

Reply:
Yes, thats exported for usage. Its exported to be used in cloudwatch agent, which is the purpose of this CR